configure

```diff
@@ -793,6 +795,13 @@ EOF
     recheck_use nas
 fi
 
+if [ "$use_sdl" != no ]; then
+    remember_use sdl
+    check_include SDL2/SDL.h
```

Comment:
```suggestion
    check_include SDL2/SDL.h "" "#define SDL_MAIN_HANDLED"
```
check_include fails because of a linker error (undefined reference to WinMain). This happens because SDL wraps the main function and renames the original, however it doesn't seem to substitute its own in this case. See https://stackoverflow.com/questions/32342285/undefined-reference-to-winmain16-c-sdl-2/32343111#32343111
`#define SDL_MAIN_HANDLED` prevents tinkering with the main function and will allow compilation as-is.